### PR TITLE
Direct users to milli for the core library in the README

### PR DESCRIPTION
**Pull request by okyanusoz**

### CODE REVIEW

**Comment by curquiza** [comment]
Hello @okyanusoz! thanks for the PR. I would rather ask for a review of our tech writing team before any merge, but they have a lot on their plate right now.
So, sorry for the delay, but the review will come :) We'll come back to you

**Comment by okyanusoz** [comment]
No problem.

File: README.md [hunk 1]
```diff
@@ -171,6 +171,10 @@ Now that your MeiliSearch server is up and running, you can learn more about how
 
 Hey! We're glad you're thinking about contributing to MeiliSearch! However, we are currently working on a huge refactor and accepting PRs on this repository wouldn't be productive. We are sorry about this! Be sure we are doing our best so that you can contribute to MeiliSearch again as soon as possible ❤️
 
+## Core library
+
+This repository does not contain MeiliSearch's indexing and searching code. For that, see [the milli repository](https://github.com/meilisearch/milli), which serves as the core for MeiliSearch.
```
Comment (curquiza) [inline README.md:176]:
```suggestion
## Code base

To perform search and indexation, MeiliSearch uses [the `milli` library](https://github.com/meilisearch/milli), present in another repository. This library only works for one index. <br>
The MeiliSearch code base available in this repository manages multiple indexes handling, the update store, and the HTTP API.

Also, the tokenization is handled by [the`tokenizer` library](https://github.com/meilisearch/tokenizer/).
```

Comment (curquiza) [reply README.md:176]: WDYT @Kerollmops and @guimachiavelli? 😇

Comment (Kerollmops) [reply README.md:176]: > and the HTTP API.

+ serve a pretty HTTP Dashboard API?

**Comment by curquiza** [comment]
@guimachiavelli, when you have time to check this 😇

File: README.md [hunk 2]
```diff
@@ -171,6 +171,13 @@ Now that your MeiliSearch server is up and running, you can learn more about how
 
 Hey! We're glad you're thinking about contributing to MeiliSearch! However, we are currently working on a huge refactor and accepting PRs on this repository wouldn't be productive. We are sorry about this! Be sure we are doing our best so that you can contribute to MeiliSearch again as soon as possible ❤️
 
+## Code base
+
+To perform search and indexation, MeiliSearch uses [the `milli` library](https://github.com/meilisearch/milli), present in another repository. This library only works for one index. <br>
+The MeiliSearch code base available in this repository manages multiple indexes handling, the update store, the HTTP API, and serves a pretty HTTP dashboard.
+
+Also, the tokenization is handled by [the `tokenizer` library](https://github.com/meilisearch/tokenizer/).
+
```
Comment (guimachiavelli) [inline README.md:180]: What do you think about this? I want to focus first on what this specific repository is for before linking to the other bits that people might be interested in.

```suggestion
## Core engine and tokenizer

The code in this repository is only concerned with managing multiple indexes, handling the update store, and exposing an HTTP API.

Search and indexation are the domain of our core engine, [`milli`](https://github.com/meilisearch/milli), while tokenization is handled by [our `tokenizer` library](https://github.com/meilisearch/tokenizer/).
```

**Review by Kerollmops — APPROVED** [review]
LGTM!
bors merge

**Comment by curquiza** [comment]
Merging without bors since clippy errors are fixed on `release-v0.21.0` and does not concern this PR.

**Pull request merged**